components/server-side-render/test/index.js

```diff
@@ -0,0 +1,64 @@
+import httpBuildQuery from 'http-build-query';
```

Comment:
Also why are we testing an external library?

Reply:
@youknowriad https://github.com/WordPress/gutenberg/pull/7339#issuecomment-398066370

Reply:
Better implementation in #7371